zebra-rpc/qa/rpc-tests/feature_nu6.py

```diff
@@ -27,12 +27,15 @@ def __init__(self):
         self.cache_behavior = 'clean'
 
     def setup_network(self):
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir)
+        # Add pre and post NU6 funding streams to the node.
+        args = [True]
```

Comment:
Subjective/Nitpick: It would be nice to name this `extra_args` and move it to the line immediately after `self.num_nodes = 1`.

Nitpick: Typo on line 21.